pipeline_manager/src/config.rs

```diff
@@ -110,6 +110,24 @@ pub(crate) struct ManagerConfig {
     #[serde(skip)]
     #[arg(short, long)]
     pub config_file: Option<String>,
+
+    /// [Developers only] Inject a SQL file into the database when starting the
+    /// manager.
+    ///
+    /// This is useful to populate the DB with state for testing.
+    #[serde(skip)]
+    #[arg(short, long)]
+    pub initial_sql: Option<String>,
+
+    /// [Developers only] Run in development mode.
+    ///
+    /// This runs with permissive CORS settings and allows the manager to be
+    /// accessed from a different host/port.
+    ///
+    /// The default is `false`.
+    #[serde(default)]
+    #[arg(long)]
+    pub dev_mode: bool,
```

Comment:
How's this different from binding it to 0.0.0.0 instead of 127.0.0.1?

Reply:
Let me check tomorrow if 0.0.0.0 makes a difference. I ran into a problem because the browser prevents the UI running on port X making requests to the manager running on port Y (you can enable it by configuring [CORS](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS) correctly).

Reply:
I think I get it now, instead of the flag you want to compute it e.g., `dev_mode = host == 'localhost'`?

Reply:
> Let me check tomorrow if 0.0.0.0 makes a difference. I ran into a problem because the browser prevents the UI running on port X making requests to the manager running on port Y (you can enable it by configuring [CORS](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS) correctly).

Oh, because the UI isn't currently hosted by the server.

> I think I get it now, instead of the CLI flag you want to compute it e.g., dev_mode = host == 'localhost'?

No, that's not what I meant. It makes sense now that I know what you're doing.